### PR TITLE
docs/modules: add initial short explanation about modules

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: docs/software_architecture.md [hunk 1]
```diff
@@ -4,15 +4,27 @@ adhocracy+ data model
...
-## Modules and Phases - Adhocracy4
-The definition and descriptions of modules and phases used in Adhocracy4
-can be found in the docs
+The definition and descriptions of modules and phases can be found
 [here](https://github.com/liqd/adhocracy4/blob/master/docs/phases_and_modules.md).
 
+## Module example in adhocracy+
+An example of a module would be the agenda setting. The description and phases
+used are defined
+[here in the blueprints](./blob/92849ba0d8eb5a27c1571f319be0356db8e07347/apps/dashboard/blueprints.py#L40).
```
Comment (Rineee) [inline docs/software_architecture.md:19]: The links dont work. But is that maybe because its not yet linking from master?

Comment (fuzzylogic2000) [reply docs/software_architecture.md:19]: Oh, yeah, I might have deleted too much of the link. Will fix that!

Comment (fuzzylogic2000) [reply docs/software_architecture.md]: I only managed to do the links with absolute ones. :woman_shrugging:

**Review by Rineee — COMMENTED** [review]
Cool, works now!

**Pull request merged**